cpp/merge_print_and_online.cc

```diff
@@ -272,13 +272,12 @@ void EliminateDanglingOrUnreferencedCrossLinks(const bool debug, const std::unor
 }
 
 
-const std::set<MARC::Tag> LINK_TAGS{ "800", "810", "830", "773", "775", "776" };
+const std::set<MARC::Tag> LINK_TAGS{ "800", "810", "830", "773", "830" };
```

Comment:
Duplicate entry "830". Also, is it corect that 775 and 776 have been removed?

Reply:
Done